### PR TITLE
Deploy app

### LINKED ISSUE #2: Deploy app
Replace database with production database
Configuration etc. for GCP or AWS
Deploy ASAP; deploy updates and new features as soon as they are ready (like they work and have tests)

### PR DESCRIPTION
**Pull request by cjdufault**
@yp0975eu @NateBigStone @ashley7295 @sunnycake 
[Deployed app to Google App Engine](https://lmnop-295416.uc.r.appspot.com/)
Closes #2

### CODE REVIEW

**Comment by yp0975eu** [comment]
This commit history shows the Postgress DB password: https://github.com/cjdufault/lmn/pull/19/commits/755e8c78394882f0a4593b7cf938c454ae93f85c

Also, it looks like nothing has changed in models.py, but it is showing newlines. Do you know what happened here? https://github.com/cjdufault/lmn/pull/19/files#diff-dcfeaa5ac3dffd54264034760914588d92fc7395c81eaa9274715840dc28100b

File: app.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+handlers:
+- url: /static
+  static_dir: www/static/
+
+- url: /.*
+  script: auto
+
+env_variables:
+  POSTGRE_PW: iKlF9eCqJpILnHbE
```
Comment (yp0975eu) [inline app.yaml]: This commit history shows the Postgress DB password :(

**Review by yp0975eu — CHANGES_REQUESTED** [review]
Can you clean up the commit history? Maybe rebase the commits so that it doesn't show the DB PW.

File: app.yaml (hunk 1 above)
Comment (NateBigStone) [reply app.yaml]: It could be more trouble than it's worth to delete branches/ squash/ rebase/ chop off heads/ etc. Maybe just change the pw?

**Comment by cjdufault** [comment]
Sorry about that, that was sloppy of me. I changed the db password.

**Review by yp0975eu — APPROVED** [review]

**Review by NateBigStone — APPROVED** [review]

**Pull request merged**